spec/features/food/food_new_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'New Food', type: :system do
+  before(:each) do
+    RecipeFood.delete_all
+    Food.delete_all
+    Recipe.delete_all
+    User.delete_all
+    @user = User.create(name: 'User', surname: 'Spec', email: 'testing@gmail.com', password: 'testing')
+    @r1 = Recipe.create(user: @user, name: 'Recipe one', preparation_time: 1, cooking_time: 2,
+                        description: 'Recipe one description', public: true)
+    @f1 = Food.create(user: @user, name: 'Food item', measurement_unit: 'kg', price: 2, quantity: 5)
+  end
+
+  context 'Adds new Food item' do
+    it 'Instance to the table' do
+      visit new_user_session_path
+      fill_in 'Email', with: @user.email
+      fill_in 'Password', with: @user.password
+      click_button 'Log in'
+      sleep(1)
```

Comment:
- [ ] Some test cases are failing, Please ensure that all test cases are passing.

![image](https://github.com/the-faizmohammad/Food_reciepes/assets/68796232/3f893b2c-b62e-4122-96f0-19ce9e0a4be6)

Reply:
![Screenshot 2024-02-16 200153](https://github.com/the-faizmohammad/Food_reciepes/assets/134736318/32979f77-4d7a-48e8-9c0b-4281b9462402)